tests/test_meshtastic_utils_message_edge.py

```
@@ -0,0 +1,1280 @@
+            "from": TEST_PACKET_FROM_ID,
+            "to": 987654321,
+            "channel": 0,
+            "id": TEST_PACKET_ID,
+            "rxTime": TEST_PACKET_RX_TIME,
+            # No 'decoded' field
+        }
+
+        import inspect
+        from concurrent.futures import Future
+
+        def _done_future(coro, *args, **kwargs):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_done_future`

(ANN202)